mkdocs/docs/configuration.md

```diff
@@ -228,19 +228,19 @@ catalog:
 catalog:
   default:
     uri: thrift://localhost:9083
-    s3.endpoint: http://localhost:9000
-    s3.access-key-id: admin
-    s3.secret-access-key: password
+    hive:
+      hive2-compatible: true
+      use-kerberos: true
 ```
 
-When using Hive 2.x, make sure to set the compatibility flag:
+<!-- markdown-link-check-disable -->
 
-```yaml
-catalog:
-  default:
-...
-    hive.hive2-compatible: true
-```
+| Key                   | Example | Description                       |
+| --------------------- | ------- | --------------------------------- |
+| hive.hive2-compatible | true    | Using Hive 2.x compatibility mode |
+| hive.use-kerberos     | true    | Using authentication via Kerberos |
```

Comment:
How about adding authorization to the property? For those that are not familiar with Kerberos, they know that it is related to auth
```suggestion
| hive.kerberos-authorization     | true    | Using authentication via Kerberos |
```

Reply:
That's better name! I will changed the key to `kerberos-authorization` then